tests/snmpsim/netscaler-sdx.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.5951.6
```

Comment:
Absolutely, just don't chop it down, i.e replace domain.com with example.com - don't remove it.